test/Dialect/Comb/lower-comb.mlir

```diff
@@ -0,0 +1,16 @@
+// RUN: circt-opt %s --lower-comb | FileCheck %s
+// RUN: circt-opt %s --lower-comb=assign-to-wire=true | FileCheck %s --check-prefix=WIRE
+
+// CHECK-LABEL: hw.module @err
```

Comment:
It looks to me like the lowering supports a case where there are no input bits (and the table size = 1).  Does it?  If so, add a test? I've found it is convenient to support this degenerate case.